hangman_lunch_final.rb

```diff
@@ -0,0 +1,145 @@
+require "colorize"
+class Hangman #
+
+  def initialize
+    # hangman board:
+    @first        = "     ______       "
+    @second       = "    |      |      "
+    @third        = "           |      "
+    @fourth       = "           |      "
+    @fifth        = "           |      "
+    @sixth        = "           |      "
+    @seventh      = "    _______|___   "
+    @eighth       = "   |           |  "
+    @ninth        = "                  "
+    @tenth        = "  H A N G M A N   " # change based on how many letters, etc
+
+    start
+  end
+
+  def display_board
+    @hangman = [@first, @second, @third, @fourth, @fifth, @sixth, @seventh, @eighth, @ninth, @tenth]
+    @hangman.each do |line|
+      puts line
+    end
+    puts "\n"
+  end
+
+  def set_answer
+    possible_answers = ["potato", "broccoli", "animal", "hotdog",
+    "elephant", "cat", "taco", "whale", "walrus", "vulture"]
+    # select random word from possible anaswer
+    which_random_index = rand(0...possible_answers.length)
+    @answer = possible_answers[which_random_index]
```

Comment:
I changed it so the possible_answers is now a constant and I implemented the `sample` method on it.. Thanks